de/nmichael/efa/gui/util/EfaMenuButton.java

```diff
@@ -174,6 +175,12 @@ public boolean isSeparator() {
                     International.getStringWithMnemonic("Backups"),
                     BaseFrame.getIcon("menu_backup.png")));
         }
+        if (admin == null || admin.isAllowedCreateBackup()) {
```

Comment:
Instead of backup permission, should be maybe check for isAllowedAdministerProjectLogbook()?